docs/ref/contrib/sitemaps.txt

```diff
@@ -173,13 +173,13 @@ Note:
 
         **Optional.** Either a method or attribute.
 
-        If it's a method, it should take one argument -- an object as returned by
-        :attr:`~Sitemap.items()` -- and return that object's last-modified date/time, as a Python
-        ``datetime.datetime`` object.
+        If it's a method, it should take one argument -- an object as returned 
```

Comment:
I merged these line wrapping changes in a496d10a8c3b8b3080f0daa7348b33aed790a967 since it's better not to include unrelated cleanups in a feature like this.